docs/insomnia/migrate-from-designer.md

```diff
@@ -35,28 +39,44 @@ If there is an unexpected error during migration, you will be shown the error th
 
-In order to manually restore:
+In order to manually restore, do the following:
 
 1. Close Insomnia
-2. Delete the plugins, responses, and version-control folders in the <app-data-dir>/insomnia live data directory
-3. Copy-paste everything from <app-data-dir>/insomnia/core-backup into <app-data-dir>/insomnia, overwriting everything that exists
+2. Delete the plugins, responses, and version-control folders in the `<app-data-dir>/insomnia` live data directory
+3. Copy-paste everything from `<app-data-dir>/insomnia/core-backup` into `<app-data-dir>/insomnia`, overwriting everything that exists
 4. Launch Insomnia
 
-## Edge cases
+## Edge case
+
+If you moved your workspace from Core to Designer several months ago, and are now merging the same workspace (with the same id) from Designer back into Core, this edge case applies. The merge won't remove any children of the workspace (such as requests, request groups, certificates), but if a duplicate id is found, it will favor the data coming from Designer.
+
+In the unlikely scenario that you can't see your requests or workspace in Insomnia after migration, [restore your data from your backup](#restore-from-backup). Alternatively, because the migration is primarily additive and will not remove anything from the existing database (it will overwrite if the same id is found), some entries may become orphaned. Find orphaned entries by opening the `insomnia.{model}.db` files.
+
+If you have orphaned entries, or are not seeing data you expect to see when you launch Insomnia, send an email to [Support](mailto:support@insomnia.rest) outlining your steps, including your application logs, and the team will assist where possible.
+
+## Migration Alternatives
+
+You can still migrate your data in the following situations:
 
-There are potential edge cases where a user moved their workspace from Core to Designer several months ago, and is now merging the same workspace (with the same id) from Designer back into Core. This merge will not remove any children of the workspace (eg. requests, request groups, certificates, etc), but if a duplicate id is found it will favor the data coming from Designer.
+* You've never installed Insomnia but I have Insomnia Designer
+* You have Insomnia 2021.7.3 or newer installed already
 
-In the unlikely scenario where you cannot see your requests or workspace in Insomnia after migration, you can try to restore from the backup as described above. Alternatively, because the migration is primarily additive and will not remove anything from the existing database (it will overwrite if the same id is found as per the previous paragraph), some entries may have become orphaned. It takes some digging, but you can find orphaned entries by opening the insomnia.{model}.db files.
+{:.alert .alert-warning}
+**Warning**:  We use the same data directory for all versions, so data you attempt to migrate to a newer version may be fully or partially incompatible
```

Comment:
This is an interesting comment and I'd like to understand the inspiration for it 🤔 

The data is always forwards compatible (going from an older version to a newer version), because the data is transformed to whatever is needed by the version being used. We just don't have backwards compatibility (downgrading to an older version).

Reply:
Interesting, this is worth a team chat as I have heard differently! Let's do that prior to release.

Reply:
it's gonna be tough to describe this: but I think the notion is:

1. if you are in insomnia 2022.1.0, let's say
2. and you also have insomnia designer that you haven't migrated yet
3. and you downgrade insomnia to 2021.7.2 in order to begin the migration

you will have found yourself in a bit of a pickle.  I think we can mention that downgrading insomnia is not generally supported, and if they chose to do that (such as in the example above) then we suggest they make a backup of their data directory first, do the migration, then export everything to a file, then upgrade and replace the backup -> at which point they can now import the migrated stuff from the file they exported.   It's difficult to explain without getting complicated, but so it is.

Reply:
I think there are a couple of things at play here and maybe we're conflating the term "migrate". 
- Upgrading the app should always be supported from a previous version. To Opender's point, this is "forward compatible." No surprises there. Occasionally, the data needs to be transformed to match how the new app will operate.
- Migrating data from Insomnia Designer will no longer work after this next release. In order to migrate, you must use an older version of Insomnia.
- Downgrading Insomnia has undefined behavior with regards to your data (read: may catastrophically break your data). 

From what I can tell, we want to tell people not to downgrade or at least to backup their data first.

Reply:
> * if you are in insomnia 2022.1.0, let's say
> 
> * and you also have insomnia designer that you haven't migrated yet
> 
> * and you downgrade insomnia to 2021.7.2 in order to begin the migration

This rationale makes sense 👍🏽 the current text I commented on is IMO way too generic to describe this specific/edge case where a user might need to downgrade. If I as a user read that warning, I'd be pretty afraid to even upgrade (let alone downgrade).

> From what I can tell, we want to tell people not to downgrade or at least to backup their data first.

And this; I think we do (or if we don't, we should) say somewhere (not on this page) that downgrades are not supported. But on this page, because a user may need to downgrade, tell them to manually back everything up.

Reply:
how about something like:

```
**Warning**:  Although Insomnia Designer uses a separate data directory, Insomnia uses the same data directory for all versions.  Any time you attempt to migrate from Insomnia Designer you should backup both data directories first.  This is especially important if it means you have to (temporarily) downgrade Insomnia from a previously installed newer version.
```

Reply:
🔥